ci/pipeline

```diff
@@ -246,36 +279,78 @@ def jenkins(): Unit = {
  * @return Version and artifact description of Marathon build.
  */
 @main
-def release(requestVersion: String, gitSha: String, latest: Boolean = false): Unit = {
-  val releaseVersion = withoutVersionPrefix(requestVersion)
-  val tagVersion = s"v$releaseVersion"
+def release(args: String*): Unit = {
+  case class Config(
```

Comment:
I like the introduction of scopt. I thought Ammonite could already parse arguments, though.

Reply:
You're right, it can already parse arguments. Support is pretty lacking.

Take the following example:

```

13:05 $ cat args.sc
@main def main(run_tests: Boolean, rest: String*): Unit = {
  println(run_tests)
  println(rest.mkString(","))
}
 ~
13:06 $ amm args.sc --flag true lol 1 2 3
The following argument failed to parse:

--run_tests: Boolean = "lol" failed to parse with java.lang.IllegalArgumentException: 'lol' is not a boolean.

expected signature:

main
  --run_tests  Boolean
  --rest       String*
```

In this case, since I provided the wrong argument name, it assumed the next argument, "LOL", was for the perimeter `–-run_tests`.

This can be worse if you accidentally mistype a parameter:

```
13:09 $ amm args.sc true --run_tests false 1 2 3 --lol false
true
--run_tests,false,1,2,3,--lol,false
 ~
```

Ammonite just completely ignores it.

So, while it would be possible to use Ammonite's argument parsing, there are more pitfalls and the potential for surprise is much greater.

Other limitations of Ammonite's argument pursing abilities is that you cannot specify default values for arguments.

Reply:
Thanks for clarifying!